src/darwin/mod.rs

```diff
@@ -75,41 +75,31 @@ impl Descriptor {
         let cg_font = ct_font.copy_to_CGFont();
 
         let fallbacks = if load_fallbacks {
-            descriptors_for_family("Menlo")
-                .into_iter()
-                .find(|d| d.font_name == "Menlo-Regular")
-                .map(|descriptor| {
-                    let menlo = ct_new_from_descriptor(&descriptor.ct_descriptor, size);
-
-                    // TODO fixme, hardcoded en for english.
-                    let mut fallbacks = cascade_list_for_languages(&menlo, &["en".to_owned()])
-                        .into_iter()
-                        .filter(|desc| !desc.font_path.as_os_str().is_empty())
-                        .map(|desc| desc.to_font(size, false))
-                        .collect::<Vec<_>>();
-
-                    // TODO, we can't use apple's proposed
-                    // .Apple Symbol Fallback (filtered out below),
-                    // but not having these makes us not able to render
-                    // many chars. We add the symbols back in.
-                    // Investigate if we can actually use the .-prefixed
-                    // fallbacks somehow.
-                    if let Some(descriptor) =
-                        descriptors_for_family("Apple Symbols").into_iter().next()
-                    {
-                        fallbacks.push(descriptor.to_font(size, false))
-                    };
-
-                    // Include Menlo in the fallback list as well.
-                    fallbacks.insert(0, Font {
-                        cg_font: menlo.copy_to_CGFont(),
-                        ct_font: menlo,
-                        fallbacks: Vec::new(),
-                    });
-
-                    fallbacks
-                })
-                .unwrap_or_else(Vec::new)
+            // Use Menlo as an original font.
+            let font_name = match (
+                ct_font.symbolic_traits().is_bold(),
+                ct_font.symbolic_traits().is_italic(),
+            ) {
+                (true, true) => "Menlo-BoldItalic",
+                (true, _) => "Menlo-Bold",
+                (_, true) => "Menlo-Italic",
+                _ => "Menlo-Regular",
+            };
```

Comment:
Wait, did you change back to the old version again? Why?

Reply:
As I have said previously, I think that changing the source fallback font from Menlo to a current font is out of this PR's purpose. The purpose is to change then only fallback font style. But as you said previously, it makes sense in general and I think so too. So I will send another PR to do it.

Should we do it in this PR? If so, I will do it again and add the change to changelog.

Reply:
> Should we do it in this PR? If so, I will do it again and add the change to changelog.

Honestly I don't mind either way, I'll just write the commit message to capture the purpose of the change accordingly.

But if you want to send a separate PR, I'd like to merge that first rather than merging a shitty version of this PR just to later fix it all up again.

Reply:
I got it. I'll change an original fallback font from Menlo-Regular to the current font again in this PR and write a commit message and CHANGELOG about the change.

Reply:
Done.